### PR TITLE
Add answer visibility toggle button

### PR DESCRIPTION
**Pull request by vyshnavsdeepak**

### CODE REVIEW

**Comment by vyshnavsdeepak** [comment]
#5

**Comment by vyshnavsdeepak** [comment]
https://github.com/knadh/dont.build/issues/5

File: index.html [hunk 1]
```diff
@@ -49,6 +49,9 @@ <h1 class="title">Don't build (or build) that feature</h1>
                 <input type="range" min="1" max="10" step="0.5" value="5.5" />
             </fieldset>
         </form>
+        <div class="text-center">
+          <button id="answer-visibility-toggle"></button>
```
Comment (knadh) [inline index.html]: This should have a default value for accessibility.

`<button id="answer-visibility-toggle">Show Answer</button>`

Comment (vyshnavsdeepak) [reply index.html]: Understood. Amending it

**Comment by vyshnavsdeepak** [comment]
Shall squash commits once you confirm

**Pull request merged**